anesthetic/plotting/_matplotlib/core.py

```diff
@@ -59,19 +79,40 @@ def __init__(self, data, x, y, s=None, c=None, **kwargs) -> None:
         super().__init__(data, x, y, s, c, **kwargs)
 
 
-class ScatterPlot2d(_CompressedMPLPlot, PlanePlot):
+class _PlanePlot2d(PlanePlot):
+
+    def _make_plot(self):
+        colors = self._get_colors()
+        print(self.data)
+        data = (
+            create_iter_data_given_by(self.data, self.kind)  # safe for now
+            if self.by is not None
+            else self.data
+        )
+        x = data[self.x].values
+        y = data[self.y].values
+        ax = self._get_ax(0)  # another one of these hard-coded 0s
+
+        kwds = self.kwds.copy()
+        label = pprint_thing(self.label)
+        kwds["label"] = label
+
+        style, kwds = self._apply_style_colors(colors, kwds, 0, label)
+        if style is not None:
+            kwds["style"] = style
+
+        self._plot(ax, x, y, **kwds)
```

Comment:
It looks like this code has been copied from `HistPlot`. Is there a reason (a) `HistPlot` is the right template to use (rather than `PlanePlot` or `LinePlot` and (b) why we can't just (diamond) inherit this functionality?